packages/mobile/src/screens/sign-on-screen/screens/SignOnScreen.tsx

```diff
@@ -131,15 +132,24 @@ const ExpandablePanel = (props: ExpandablePanelProps) => {
   )
 }
 
+export type SignOnScreenParams = {
+  screen: SignOnScreenType
+}
+
 /*
  * Manages the container for sign-up and sign-in flow
  * Not using navigation for this due to transition between sign-in and sign-up
  */
-export const SignOnScreen = () => {
+export const SignOnScreen = ({ route }) => {
+  const { params } = route
   const [email, setEmail] = useState('')
-  const [screen, setScreen] = useState<SignOnScreenType>('sign-up')
+  const [screen, setScreen] = useState<SignOnScreenType>(params.screen)
```

Comment:
In order to link back to sign-in from the "Create Login Details" screen (which also has an email field), I had to add a param for the screen

Reply:
smart idea, just double checking we want to link back from create-login-details?